### PR TITLE
{lib}[fosscuda/2020b] PyTorch-Geometric v2.0.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PyTorch-Geometric/PyTorch-Geometric-2.0.1-fosscuda-2020b-PyTorch-1.9.0.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    ('Python', '3.8.6'),
+    ('PyTorch', local_pytorchver),
+    ('scikit-learn', '0.23.2'),
+    ('scikit-image', '0.18.1'),
+    ('numba', '0.53.1'),
+    ('h5py', '3.1.0'),
+    ('tqdm', '4.56.2'),
+    ('RDFlib', '5.0.0'),
+    ('ASE', '3.20.1'),
+]
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/p/PyTorch-Geometric/PyTorch-Geometric-2.0.1-fosscuda-2020b-PyTorch-1.9.0.eb:23]:
```suggestion
    ('ASE', '3.20.1'),
    ('YACS', '0.1.8'),
]
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
I think we need to add `YACS`:
```
`pip check` failed:
torch-geometric 2.0.1 requires yacs, which is not installed.
```

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @Micket!

**Pull request merged**